pkg/plugin/pr-sanitizer/pr_title_description_status_service.go

```diff
@@ -0,0 +1,40 @@
+package prsanitizer
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/github"
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	"github.com/arquillian/ike-prow-plugins/pkg/scm"
+	"github.com/arquillian/ike-prow-plugins/pkg/status"
+)
+
+type prTitleDescriptionStatusService struct {
```

Comment:
The name of the struct could be shorter - eg: `prSanitizerStatusService`
same for the file: `pr_sanitizer_status_service.go`